File: mindvault-ai-python/main/python/models/to_embedding_documents_dto.py
from pydantic import BaseModel, Field


class ToEmbeddingDocumentsDTO(BaseModel):
    id: str
    user_id: str = Field(alias="userId")
    source_type: str = Field(alias="sourceType")
    title: str
    content: str
    created_at: int = Field(alias="createdAt")

File: mindvault-common/src/main/java/org/example/common/ai/JavaAndPythonContract/ToEmbeddingDocumentsDTO.java
package org.example.common.ai.JavaAndPythonContract;


import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ToEmbeddingDocumentsDTO {

    private String id;//资源id
    private String userId;//用户id
    private String sourceType;//资源类型
    private String title;//标题
    private String content;//内容
    private Integer createdAt;//创建时间
}
